js/qsm-quiz.js

```diff
@@ -1218,6 +1218,16 @@ jQuery(function() {
 		}
 	});
 
+	// Autocomplete off
+	jQuery(document).ready(function(){
```

Comment:
we are already on ready why do we need to add ready again?

Reply:
Ok I will check and fix it.

Reply:
I have fixed it.